src/twisted/internet/iocpreactor/abstract.py

```diff
@@ -329,6 +329,8 @@ def loseConnection(self, _connDone=failure.Failure(main.CONNECTION_DONE)):
         """
         Close the connection at the next available opportunity.
 
+        This is the same code from twisted.internet.abstract.FileDescriptor.loseConnection
```

Comment:
What is the point of this element of the docstring? What do you expect a user to do with it?  Should it be a maintenance note (in a comment) mentioning that this should be deduplicated?